data/_types/lists/index.json

```diff
@@ -0,0 +1,167 @@
+{
+  "layout": {
+    "template": "layout"
+  },
+  "meta": {
+    "title": "Lists - Types - UK Parliament",
+    "request-id": "|Pk5Jq3rkRal4MA==.",
+    "open-graph": {
+      "title": "Lists - Types - UK Parliament",
+      "original-url": "https://localhost/",
+      "image-url": "https://static.parliament.uk/assets-public/opengraph-oblong.png",
+      "image-width": "1200",
+      "image-height": "630",
+      "twitter-card": "summary_large_image"
+    },
+    "opensearch-description-url": "https://localhost/search/opensearch"
+  },
+  "header-components": [
+    {
+      "name": "link",
+      "data": {
+        "link": "#content",
+        "display": {
+          "name": "partials__display",
+          "data": [
+            {
+              "component": "skip-to-content"
+            }
+          ]
+        },
+        "selector": "skiplink",
+        "content": "shared.header.skip-to-content"
+      }
+    },
+    {
+      "name": "header",
+      "data": {
+        "components": [
+          {
+            "name": "link",
+            "data": {
+              "link": "/",
+              "display": {
+                "name": "partials__display",
+                "data": [
+                  {
+                    "component": "uk_parliament"
+                  }
+                ]
+              },
+              "label": "shared.header.label",
+              "components": [
+                {
+                  "name": "icon__uk-parliament",
+                  "data": "shared.header.label"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    }
+  ],
+  "main-components": [
+    {
+      "name": "section__primary",
+      "data": {
+        "content-flag": true,
+        "components": [
+          {
+            "name": "heading1",
+            "data": {
+              "heading": {
+                "content": "Lists"
+              }
+            }
+          }
+        ]
+      }
+    },
+    {
+      "name": "section__section",
+      "data": {
+        "display": {
+          "name": "partials__display",
+          "data": [
+            {
+              "component": "section",
+              "variant": "major"
+            },
+            {
+              "component": "section",
+              "variant": "wide"
+            }
+          ]
+        },
+        "components": [
+          {
+            "name": "status__highlight",
+            "data": {
+              "display": {
+                "name": "partials__display",
+                "data": [
+                  {
+                    "component": "status",
+                    "variant": "highlight"
+                  }
+                ]
+              },
+              "components": [
+                {
+                  "name": "paragraph",
+                  "data": [
+                    {
+                      "content": "There is currently no prototypes to show on this page."
```

Comment:
Change to something more generic to use everywhere. 'There are currently no views'